optuna/visualization/parallel_coordinate.py

```diff
@@ -7,21 +7,18 @@
 from optuna.visualization.utils import _check_plotly_availability
 from optuna.visualization.utils import is_available
 
-if type_checking.TYPE_CHECKING:
-    from typing import DefaultDict  # NOQA
-    from typing import List  # NOQA
-    from typing import Optional  # NOQA
-
-    from optuna.study import Study  # NOQA
+from typing import DefaultDict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+from optuna.study import Study  # NOQA
 
 if is_available():
     from optuna.visualization.plotly_imports import go
 
 logger = get_logger(__name__)
 
 
-def plot_parallel_coordinate(study, params=None):
-    # type: (Study, Optional[List[str]]) -> go.Figure
+def plot_parallel_coordinate(study: Study, params: Optional[List[str]] = None) -> 'go.Figure':
```

Comment:
I encountered the problem of importing `Study` class.
So, I wonder if you should use a forward reference here. What do you think about this?
https://github.com/optuna/optuna/issues/711#issuecomment-597088465

Reply:
I think the revision above can not pass flake8 CI test....

Reply:
Ah, you're right.
You have to import `optuna` or `optuna.study` at the head of `parallel_coordinate.py`.

Reply:
And I found it doesn't need to use forward reference since there is no circular importing here.
You can use `from optuna.study import Sutdy` and `Study` here, really sorry for confusing you. :bow: